src/components/calculator/button/Button.css

```diff
@@ -0,0 +1,32 @@
+.buttonBox {
+  width: 100%;
+  height: calc(100% - 110px);
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  grid-template-rows: repeat(5, 1fr);
+  grid-gap: 10px;
+}
+
+button {
+  border: none;
+  background-color: rgb(80, 60, 209);
+  font-size: 24px;
+  color: rgb(255, 255, 255);
+  font-weight: bold;
+  cursor: pointer;
+  border-radius: 10px;
+  outline: none;
+}
+
+button:hover {
+  background-color: rgb(61, 43, 184);
+}
+
+.equals {
+  grid-column: 3 / 5;
+  background-color: rgb(243, 61, 29);
+}
+
+.equals:hover {
+  background-color: rgb(228, 39, 15);
+}
```

Comment:
- [ ] Your design is really nice but it would be nice if your design match the original design given. You are required to match your design to the one given in the project brief. Please follow the instruction guides and later in your free time after this module, you can add features and style to all your apps to match your taste but it's good to follow the projects instructions given as they help guide you better into becoming a professional and better developer that write a simple and understandable code. This will help you better work in a remote environment 👍 

**The original design**
![image](https://user-images.githubusercontent.com/50941074/176323862-4f97d0ff-674a-4732-bd97-990080b23be6.png)

**Your Design**
![image](https://user-images.githubusercontent.com/50941074/176324671-d785db2e-fb84-4f5d-abd3-c186830b3d6c.png)